README.md

```diff
@@ -48,10 +48,14 @@ As a developer transitioning from Rider to Neovim, I found myself missing the si
 11. [EntityFramework](#entityframework)
     - [Database](#database)
     - [Migrations](#migrations)
-12. [Nvim-dap configuration](#nvim-dap-configuration)
+12. [Language injections](#language-injections)
+    - [Showcase](#showcase)
+    - [Requirements](#requirements-3)
```

Comment:
```suggestion
    - [Requirements](#requirements)
```

Reply:
The link in TOC needs an index because we now have 2 "requirements" headers in the readme.

and I think this is a bug in my markdown file previewer in nvim, which I used to update the readme. Because in it that header was requirements-3 and it worked, but in the github preview it's requirements-2 (which makes sense).  Here is the link that leads to the correct requirements header in my fork on github: https://github.com/molostovvs/easy-dotnet.nvim/tree/feat/tree-sitter-injections?tab=readme-ov-file#requirements-2